### PR TITLE
Adds combat gloves plus, gives them to nukies.

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
Adds combat gloves that teach you krav maga. Adds these gloves to the nukie uplink for 5tc

## Why It's Good For The Game
Gives nukies more variety in things that they can buy. Isn't too OP because it cannot be combined with gloves of the northstar. Makes fighting as/against nukies more varied. Also allows nukies to run fun gimmicks or as a backup weapon by running round leg sweeping people and stomping them to death.

## Changelog
:cl:
add: adds combat gloves plus, combat gloves that teach the wearer krav maga. Gives these gloves to nukies for 5tc.
/:cl:

### CODE REVIEW

**Comment by Dashsora** [comment]
Five TC is a bit cheap, would suggest seven or eight. The damage is still pretty good, and you can silence people for insane infiltration teams.

Does krav work with cqc ?

**Comment by Sneedstar** [comment]
This is cool! Though something to keep in mind is that you may be opting out of CQC if you use krav maga as it overrides other martial arts.

**Comment by Sneedstar** [comment]
> Five TC is a bit cheap, would suggest seven or eight. The damage is still pretty good, and you can silence people for insane infiltration teams.
> 
> Does krav work with cqc ?

Krav as far as I know, overrides all other arts.

**Comment by hal9000PR** [comment]
Yeah you can only have 1 martial art at a time. I could add a warning in the uplink saying "these are incompatible with CQC"

**Comment by Sneedstar** [comment]
> Yeah you can only have 1 martial art at a time. I could add a warning in the uplink saying "these are incompatible with CQC"

I think it would be a good note to add if you could, people might go: "BA HA! I HAVE MANY MARTIAL ARTS AND SHALL BECOME MARTIAL ART CHAMPION!" or something silly along those lines.

**Comment by Aracino** [comment]
> Five TC is a bit cheap, would suggest seven or eight. The damage is still pretty good, and you can silence people for insane infiltration teams.


To be fair, compared to most martial arts, krav is pretty weak, and doens't use any combos. I believe all it does is give you bonus unarmed damage, along with the legsweep, mute punch, and lung punch abilities which are all not too strong, especially given there are better options for nukies to get. This just makes them a bit more robust than your average greytide if they find themselves with no weapons, and again, they could also get CQC which is way better in general albeit more costly.

**Comment by DeveloperLoser** [comment]
> > Five TC is a bit cheap, would suggest seven or eight. The damage is still pretty good, and you can silence people for insane infiltration teams.
> 
> To be fair, compared to most martial arts, krav is pretty weak, and doens't use any combos. I believe all it does is give you bonus unarmed damage, along with the legsweep, mute punch, and lung punch abilities which are all not too strong, especially given there are better options for nukies to get. This just makes them a bit more robust than your average greytide if they find themselves with no weapons, and again, they could also get CQC which is way better in general albeit more costly.

Yea it gives 10 bonus un armed damage, so you're hitting 15-19 brute each swing

**Comment by Denthamos** [comment]
Five TC should be fine considering the basic nature of Krav.  Its 3 ability uses all have cooldowns attached to them as well as requiring constantly activation via buttons.  Cost can always be increased later if it appears too cheap.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -952,6 +952,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/toy/carpplushie/dehy_carp
 	cost = 2
 
+/datum/uplink_item/stealthy_weapons/combat_plus
+	name = "Combat Gloves Plus"
+	desc = "Combat gloves but with installed nanochips that teach you krav maga when worn, great as a cheap backup weapon. Warning, the nanochips will override any other fighting styles such as CQC"
```
Comment (dearmochi) [inline code/datums/uplink_item.dm:957]:
```suggestion
	desc = "Combat gloves but with installed nanochips that teach you Krav Maga when worn, great as a cheap backup weapon. Warning, the nanochips will override any other fighting styles such as CQC."
```

File: code/modules/martial_arts/krav_maga.dm [hunk 2]
```diff
@@ -132,3 +132,18 @@
...
 	item_state = "fightgloves"
+
+/obj/item/clothing/gloves/color/black/krav_maga/combat // for nukies
+	name = "combat gloves plus"
+	desc = "These combat gloves have been upgraded with nanochips that teach the wearer Krav Maga"
+	icon_state = "combat"
+	item_state = "swat_gl"
+	siemens_coefficient = 0
+	permeability_coefficient = 0.05
+	strip_delay = 80
```
Comment (dearmochi) [inline code/modules/martial_arts/krav_maga.dm:143]:
```suggestion
	strip_delay = 8 SECONDS
```

File: code/modules/martial_arts/krav_maga.dm [hunk 3]
```diff
@@ -132,3 +132,18 @@
 	desc = "These gloves can teach you to perform Krav Maga using nanochips."
 	icon_state = "fightgloves"
 	item_state = "fightgloves"
+
+/obj/item/clothing/gloves/color/black/krav_maga/combat // for nukies
+	name = "combat gloves plus"
+	desc = "These combat gloves have been upgraded with nanochips that teach the wearer Krav Maga"
```
Comment (dearmochi) [inline code/modules/martial_arts/krav_maga.dm:138]:
```suggestion
	desc = "These combat gloves have been upgraded with nanochips that teach the wearer Krav Maga."
```

**Review by dearmochi — APPROVED** [review]
OK besides text nitpicks

**Review by Kyep — APPROVED** [review]

File: code/datums/uplink_item.dm [hunk 4]
```diff
@@ -952,6 +952,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/toy/carpplushie/dehy_carp
 	cost = 2
 
+/datum/uplink_item/stealthy_weapons/combat_plus
+	name = "Combat Gloves Plus"
+	desc = "Combat gloves but with installed nanochips that teach you Krav Maga when worn, great as a cheap backup weapon. Warning, the nanochips will override any other fighting styles such as CQC."
```
Comment (S34NW) [inline code/datums/uplink_item.dm:957]:
```suggestion
	desc = "Combat gloves with installed nanochips that teach you Krav Maga when worn, great as a cheap backup weapon. Warning, the nanochips will override any other fighting styles such as CQC."
```
This but isn't really needed

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**